eli5/formatters/utils.py

```diff
@@ -144,7 +144,7 @@ def tabulate(data,  # type: List[List[Any]]
 
 def format_weight(value):
     # type: (Real) -> str
-    return '{:+.3f}'.format(value)
+    return '{:+.3f}'.format(value)  # type: ignore
```

Comment:
I see that this is due to https://github.com/python/mypy/issues/3186, maybe a better fix would be to use ``Union[int, float]`` here and below instead of the ``Real`` until that issue is fixed, especially since we use ``Real`` only in this module

Reply:
I changed `Real` to `float`. No need to use `Union[float, int]` ( https://www.python.org/dev/peps/pep-0484/#the-numeric-tower ).